integration-tests/go.sum

```diff
@@ -1652,11 +1660,14 @@ github.com/smartcontractkit/chainlink-solana v1.0.2-0.20220930034647-edd5a863b87
 github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20220930034704-572ac07611cb h1:oRKhJVdoXTVQqBVSPkvfC/hxYoxsL3hldTavEuZrWOk=
 github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20220930034704-572ac07611cb/go.mod h1:FR8+xi6pmUuK6/PaH64B+OT5n/QOz7ox9pp9z05Dc2g=
 github.com/smartcontractkit/chainlink-terra v0.1.4-0.20220930034731-ef9eb53de886 h1:VVB/jdEBD9O01uOPvONEWeUVJr2/crLjVVCfTl1fOLM=
-github.com/smartcontractkit/chainlink-testing-framework v1.7.7 h1:gSj4cqd3jWoWAWQUB5rLoq4+jQ7AinZo1WixtxTMYio=
-github.com/smartcontractkit/chainlink-testing-framework v1.7.7/go.mod h1:zaCknJvXOXhYfUFnmndpoOia2+TLqRE5vfeSwawn91M=
+github.com/smartcontractkit/chainlink-testing-framework v1.7.6-0.20221031161922-6ecb37fbba9c h1:VHOqr0Gd17Msh433avpnUOwWwMuKWu/hauXGfvdGyBg=
+github.com/smartcontractkit/chainlink-testing-framework v1.7.6-0.20221031161922-6ecb37fbba9c/go.mod h1:WSxufhaN6qxWI5HbGaDeYKxXdNuA2lFoolDaoVkxtSw=
+github.com/smartcontractkit/chainlink-testing-framework v1.7.10-0.20221104150232-06964c5ad434 h1:aQwluoKv/AZbBIvDm5v6Q5DZKYa8gOpY0iY+iyl83aU=
+github.com/smartcontractkit/chainlink-testing-framework v1.7.10-0.20221104150232-06964c5ad434/go.mod h1:fV2CQJ/AWNItqkfpLxS4mWAcmRb8McbIJirAInhYlbM=
```

Comment:
you'll need to run `go mod tidy`

Reply:
Or `make gomodtidy` from the root to hit both modules.